app-libs/stf/Cargo.toml

```diff
@@ -14,9 +14,12 @@ log = { version = "0.4", default-features = false }
 sgx_tstd = { branch = "master", features = ["untrusted_fs","net","backtrace"], git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
 
 # local crates
+ita-sgx-runtime = { optional = true, default-features = false, path = "../sgx-runtime" }
 itp-hashing = { path = "../../core-primitives/hashing" }
+itp-sgx-externalities = { default-features = false, path = "../../core-primitives/substrate-sgx/externalities" }
 itp-storage = { default-features = false, path = "../../core-primitives/storage" }
 its-state = { default-features = false, optional = true, path = "../../sidechain/state" }
+sp-io = { optional = true, default-features = false, features = ["disable_oom", "disable_panic_handler", "disable_allocator"], path = "../../core-primitives/substrate-sgx/sp-io" }
```

Comment:
Should this naming really be kept to `sp-io` or maybe renamed to `itp-sp-io`?

Reply:
It can't, we cannot apply the name if we change the name.

Reply:
Since we're using it to patch the 'real' `sp-io`, I would probably leave it as-is?

Reply:
Yes, excuse my terrible phrase above, but the name needs to be the same for patches.